test/unittests/analysis_test.cpp

```diff
@@ -43,16 +41,15 @@ TEST(analysis, stack_up_and_down)
 
     ASSERT_EQ(analysis.instrs.size(), 20);
     EXPECT_EQ(analysis.instrs[0].fn, op_table[OPX_BEGINBLOCK]);
-    EXPECT_EQ(analysis.instrs[0].arg.number, 0);
     EXPECT_EQ(analysis.instrs[1].fn, op_table[OP_DUP2]);
     EXPECT_EQ(analysis.instrs[2].fn, op_table[OP_DUP1]);
     EXPECT_EQ(analysis.instrs[8].fn, op_table[OP_POP]);
     EXPECT_EQ(analysis.instrs[18].fn, op_table[OP_PUSH1]);
 
-    ASSERT_EQ(analysis.blocks.size(), 1);
-    EXPECT_EQ(analysis.blocks[0].gas_cost, 7 * 3 + 10 * 2 + 3);
-    EXPECT_EQ(analysis.blocks[0].stack_req, 3);
-    EXPECT_EQ(analysis.blocks[0].stack_max_growth, 7);
+
+    EXPECT_EQ(analysis.instrs[0].arg.block.gas_cost, 7 * 3 + 10 * 2 + 3);
```

Comment:
I'd create an alias like `auto& block = analysis.instrs[0].arg.block;` for less repetition

Reply:
Done.